devtools/cli/src/main/java/io/quarkus/cli/Encryptor.java

```diff
@@ -0,0 +1,60 @@
+package io.quarkus.cli;
+
+import java.nio.ByteBuffer;
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.SecureRandom;
+import java.util.Base64;
+import java.util.concurrent.Callable;
+
+import javax.crypto.Cipher;
+import javax.crypto.spec.GCMParameterSpec;
+import javax.crypto.spec.SecretKeySpec;
+
+import io.quarkus.cli.common.OutputOptionMixin;
+import picocli.CommandLine;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+
+@Command(name = "encryptor", header = "Encrypt Secrets using AES/GCM/NoPadding algorithm by default")
+public class Encryptor implements Callable<Integer> {
+    @Option(required = true, names = { "-s", "--secret" }, description = "Secret")
+    String secret;
+
+    @Option(required = true, names = { "-k", "--encryption-key" }, description = "Encryption Key")
```

Comment:
shouldn't this be either:
```suggestion
    @Option(required = true, names = { "-e", "--encryption-key" }, description = "Encryption Key")
```

or 

```suggestion
    @Option(required = true, names = { "-k", "--key" }, description = "Encryption Key")
```

to be consistent on naming?

Reply:
The `-e` is already reserved for `errors`. We can change it to be `--key`. The idea was to make it more descriptive.